pyproject.toml

```diff
@@ -94,12 +94,11 @@ plugins = ["pydantic.mypy"]
 [tool.ruff]
 line-length = 120
 
-
 [build-system]
 requires = ["poetry-core"]
 build-backend = "poetry.core.masonry.api"
 
 [tool.poetry.extras]
 examples=["jupyter"]
-microphone=["simpleaudio", "sounddevice"]
-legacy=["pydub"]
+microphone=["pydub", "simpleaudio", "sounddevice"]
+legacy=["pydub", "simpleaudio", "sounddevice"]
```

Comment:
apologies if we already went over this before.. why does the legacy extra need simpleaudio and sounddevice?

I would think that legacy needs pydub so it can support `VoiceSocket.send_file()`, but `simpleaudio` and `sounddevice` are only needed if you're using the microphone.

Reply:
Yep, you're exactly right - I only suggested this change in service of semantics and convenience. The legacy microphone code is, ofc, encapsulated within the legacy code, so with these changes, if a user wanted to use it they would be able to just install `"hume[legacy]"` instead of both `"hume[legacy]"` and `"hume[microphone]"`.

I can absolutely change it back though; assuming the separation of the microphone extras is kept, it also makes sense that both the current and legacy versions would need the user to install the microphone extras, instead of just one.

Reply:
No strong opinions from me. Whichever you think will lead to less confusion for users.